src/verify.c

```diff
@@ -693,19 +694,11 @@ static bool parse_options(int argc, char **argv)
 		}
 
 		if (cmdline_option_bundles) {
```

Comment:
Instead of having an extra variable can't we just have a check if bundles != NULL?

Reply:
not with the current implementation because at some point we remove everything from bundles. We could make some changes so we don't do this.

Reply:
done